makefile

```diff
@@ -6,7 +6,6 @@
 
 # The name of the binary.
 NAME = Tipster
-TARGET_DIR = Tipster
```

Comment:
The built binary was dumped into "Tipster" before which contained the source files. I moved those into "source". I'm not a fan of defining the TARGET_DIR in the make file. Without it you can have e.g. debug and release builts side-by-side without anything getting overwritten.